results-processor/wptreport.py

```diff
@@ -438,9 +448,12 @@ def prepare_labels(report, labels_str, uploader):
     # Empty labels may be generated here, but they will be removed later.
     for label in labels_str.split(','):
         labels.add(label.strip())
-    # Set the default channel to stable.
     if ('stable' not in labels) and ('experimental' not in labels):
-        labels.add('stable')
+        # Extract browser_channel into a label; default to "stable".
+        if report.run_info.get('browser_channel') in {'nightly', 'dev'}:
+            labels.add('experimental')
```

Comment:
Can you split out this change? Letting the experimental label be implied seems risky if we're going to let it mean "dev with extra features enabled".

Reply:
Thanks! I guess you usually merge for PRs where you've made the history nice?

WDYT about "Letting the experimental label be implied seems risky"?

Reply:
Ahh those are beta results. Let me add the "beta" label here.

Reply:
@foolip just updated the second commit in this PR to add the "beta" label.

Reply:
Cool! Is there an issue for adding beta icons as well?